09-lection7/01-burger-menu/burger-menu.css

```
@@ -0,0 +1,31 @@
+#burger-menu-open {
+    margin: 50px;
+    width: 24px;
+    height: 24px;
+}
+
+#burger-menu[data-open] #burger-menu-close {
+    margin: 50px;
+    width: 20px;
+    height: 20px;
+}
+
+#burger-menu:not([data-open]) #burger-menu-close {
+    display: none;
+}
+
+#burger-menu {
+    position: absolute;
+    top: 0;
+    width: 300px;
+    height: 100%;
+    background-color: lightgray;
+    transform: translateX(-309px);
```

```suggestion
    transform: translateX(-100%);
```

`translateX(-100%);` сместит элемент на свою собственную ширину. <br>
Чем больше хардкода в верстке - тем больше возможных багов при каких-то изменениях

---

согласен, вообще все как-то не красиво работало. Страница открывалась с открытым меню.